github/config.go

```diff
@@ -10,36 +10,47 @@ import (
 )
 
 type Config struct {
-	Token        string
-	Organization string
-	BaseURL      string
+	Token   string
+	Owner   string
+	BaseURL string
 }
 
-type Organization struct {
-	name        string
-	client      *github.Client
-	StopContext context.Context
+type Owner struct {
+	name         string
+	client       *github.Client
+	StopContext  context.Context
+	Organization bool
```

Comment:
How do you feel about renaming this to `IsOrganization` to signal that it's a boolean?